From 424058c86ef56a80fc9723e0afd1695681a145b8 Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Fri, 21 Apr 2017 13:31:58 +0200
Subject: [PATCH 4/5] udev: Decrease high pressure value for Elantech touchpads

The high pressure value for Elantech touchpads (both PS/2 and I2C) is
not adapted to various devices, on which the touchpad is barely usable.

Decreasing the high value makes those devices usable again, while not
introducing any major drawback for other devices.

Signed-off-by: Paul Kocialkowski <contact at paulk.fr>
---
 udev/90-libinput-model-quirks.hwdb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/udev/90-libinput-model-quirks.hwdb b/udev/90-libinput-model-quirks.hwdb
index 77d0a67..a873546 100644
--- a/udev/90-libinput-model-quirks.hwdb
+++ b/udev/90-libinput-model-quirks.hwdb
@@ -85,12 +85,12 @@ libinput:name:* Touchpad:dmi:*svnDellInc.:*
 ##########################################
 libinput:name:*ETPS/2 Elantech Touchpad*:dmi:*
  LIBINPUT_ATTR_RESOLUTION_HINT=31x31
- LIBINPUT_ATTR_PRESSURE_RANGE=24:8
+ LIBINPUT_ATTR_PRESSURE_RANGE=10:8
  LIBINPUT_MODEL_ELANTECH_TOUCHPAD=1
 
 libinput:name:*Elan Touchpad*
  LIBINPUT_MODEL_ELANTECH_TOUCHPAD=1
- LIBINPUT_ATTR_PRESSURE_RANGE=24:8
+ LIBINPUT_ATTR_PRESSURE_RANGE=10:8
 
 ##########################################
 # Google
-- 
2.12.2

